src/server/tasks/migrate.ts

```diff
@@ -121,6 +121,7 @@ const insertVideo = async (entry: MirrorEntry, mapId: number) => {
     const renderNode = 'portal2-cm-autorender';
     const requiredDemoFix = demoInfo.useFixedDemo ? FixedDemoStatus.Required : FixedDemoStatus.NotRequired;
     const demoMetadata = JSON.stringify(demoInfo.metadata);
+    const demoInputs = JSON.stringify(demoInfo.inputs);
```

Comment:
This is an old migration script that was only used for importing the data from autorender v1. Please revert the changes in this file. I'll do the migration myself because it requires access to the database.